lib/src/file_picker.dart

```diff
@@ -157,6 +157,9 @@ abstract class FilePicker extends PlatformInterface {
   /// For mobile platforms, this function will save file with [bytes] to return a path.
   /// Throws UnsupportedError on macOS if bytes are provided.
   ///
+  /// For web platform, this function will download the file with [bytes] and [fileName].
```

Comment:
```suggestion
  /// On the web, this function will start a download for the file with [bytes] and [fileName].
  /// If the bytes or filename are omitted, this will throw an ArgumentError.
```